src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -205,6 +214,13 @@ trait WorkflowComponent {
       }
     }
 
+    def optimisticLockUpdate(originalRec: WorkflowRecord): ReadWriteAction[Int] = {
```

Comment:
i didn't end up using this method in my implementation of optimistic locking (which is built directly into the raw sql). should I leave this here so it's explicit that any future methods should use optimistic locking?